uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/OptionValue.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class CRM_HRCore_Test_Fabricator_OptionValue {
+
+  private static $defaultParams = [
+    'name' => 'test option',
+    'sequential' => 1
+  ];
+
+  public static function fabricate($groupName, $params = []) {
```

Comment:
It would be better if to make this method consistent with the other fabricators which only receive the $params array (I know HRJobContract has two params, but both are arrays to be passed to the api)